public/app/features/dashboard/services/DashboardProfiler.ts

```diff
@@ -10,6 +11,27 @@ export function getDashboardSceneProfiler() {
   return dashboardSceneProfiler;
 }
 
+export function getDashboardComponentInteractionCallback(uid: string, title: string) {
+  return (e: InteractionProfileResult) => {
+    const payload = {
+      duration: e.interactionDuration,
+      networkDuration: e.networkDuration,
+      processingTime: e.interactionDuration - e.networkDuration,
```

Comment:
Let's remove this, I've introduced it but this tells nothing as there's not easy way to detect what is the actuall processing time. I'd rather integrate https://github.com/grafana/scenes/pull/1235 into interaction profiler when it lands, this will give us a more meaningful information about potential jank.